### PR TITLE
Pilgrim 1610

### PR DESCRIPTION
**Pull request by pilgrim-1610**
## Contributing

If you want to add a project to this list, please make sure that:

- [x] You read the [contributing guide](#CONTRIBUTING.md)
- [x] The project can't be just an "add your name" or simply + 1 contribution to hacktoberfest
- [x] Place them in alphabetical order
- [x] One repository per PR/contribution - _avoid conflicts and simplify review_
- [x] Your pull request is not spammy just adding an empty line, otherwise it will be marked as invalid and not count to hacktoberfest.

If they meet the requirements above, feel free to create a PR. Happy Hacking! :sparkles:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,11 +56,14 @@
 - [Killed by Google](https://github.com/codyogden/killedbygoogle)
 - [Open source drinks](https://github.com/alfg/opendrinks)
 - [Predator](https://github.com/Zooz/predator)
+-[Python Mini-Project]
```
Comment (OtacilioN) [inline README.md:59]: Can you add an empty space between the "-" and the "[" ?

Also remove the breakline, they should be in the same line

File: README.md [hunk 2]
```diff
@@ -56,11 +56,14 @@
...
+(https://github.com/Python-World/python-mini-projects)
 - [sample-programs](https://github.com/TheRenegadeCoder/sample-programs)
 - [Text Adventure Game Collection](https://github.com/D4rk3l/text-adventure-hacktoberfest)
 - [useful-dev-tools](https://github.com/lucasnaja/useful-dev-tools)
 - [Welcome to Hacktoberfest2020](https://github.com/OpenSourceTogether/Hacktoberfest-2020)
 
+
```
Comment (OtacilioN) [inline README.md:66]: Please remove this empty line.

**Review by OtacilioN — CHANGES_REQUESTED** [review]
Can you add an empty space between the "-" and the "[" ?

Also remove the breakline, they should be in the same line

File: README.md (hunk 2 above)
Comment (pilgrim-1610) [reply README.md:66]: On it

**Comment by OtacilioN** [comment]
Now that's conflicts, but I think you already solved it in a new pull request right? Can I close this one? Fell free to reopen if you fix the conflicts and wants it to get merged.

**Pull request closed without merging**